impl/proto/keytransparency_v1_service/keytransparency_v1_service.proto

```diff
@@ -70,3 +70,30 @@ service KeyTransparencyService {
     option (google.api.http) = { get: "/v1/domain/info" };
   }
 }
+
+// The KeyTransparencyBackendService API provides batch access to the directory of public keys. 
+service KeyTransparencyBackendService {
+  // BatchGetEntries returns full proofs for a set of user_ids.
+  rpc BatchGetEntries(keytransparency.v1.types.BatchGetEntriesRequest) returns (keytransparency.v1.types.BatchGetEntriesResponse) {
+    option (google.api.http) = { get: "/v1/users" };
+  }
+
+
+  // BatchUpdateEntries will put a set of UpdateEntryRequests into the queue. 
+  rpc BatchUpdateEntries(keytransparency.v1.types.BatchUpdateEntriesRequest) returns (keytransparency.v1.types.BatchUpdateEntriesResponse) {
+    option (google.api.http) = {
+      post: "/v1/users"
+      body: ""
+    };
+  }
+
+  // BatchSetEntries uses an authorized_public key to perform a set request on multiple 
+  // entries at once.  
+  rpc BatchSetEntries(keytransparency.v1.types.BatchSetEntriesRequest) returns (keytransparency.v1.types.BatchSetEntriesResponse) {
```

Comment:
What's the difference between Set and Update apis? I can see in the description that the latter accepts and the former does the work, but not why doing it all at once?